channeldb/migration25/migration.go

```diff
@@ -0,0 +1,211 @@
+package migration25
+
+import (
+	"bytes"
+	"fmt"
+
+	mig "github.com/lightningnetwork/lnd/channeldb/migration_01_to_11"
+	"github.com/lightningnetwork/lnd/kvdb"
+)
+
+var (
+	// openChanBucket stores all the currently open channels. This bucket
+	// has a second, nested bucket which is keyed by a node's ID. Within
+	// that node ID bucket, all attributes required to track, update, and
+	// close a channel are stored.
+	openChannelBucket = []byte("open-chan-bucket")
```

Comment:
We also need to migrate the `historicalChannelBucket` bucket as well. Since the `lncli closedchannel` command will read this bucket and use the _new_ routine that assumes those two new fields are there. Could've be averted (though that bucket should also be migrated) if we used a TLV there in place. I think it isn't too late to do that though (?).